docs/api/evm/api.opcode.rst

```diff
@@ -23,7 +23,7 @@ Here are some simple examples.
 
 
 The `:func:~evm.vm.opcode.as_opcode` helper
```

Comment:
@pipermerriam is it correct that all the `api.*.rst` files have been hand written so far? I think our goal would be to extract the API docs directly from the source code to not maintain them redundant or is there something else I'm missing?

Reply:
I'm fine with you defining the right path forward on this.  Historically they are probably a mix of generated and hand written.

Reply:
Yup, handwritten guides plus auto-generated API-level docs sounds good to me. :+1:

Reply:
Alright. I took a deeper look into how all this sphinx stuff works and found out about this nice `autodoc` featre where you can write directives such as `automodule`, `autoclass` etc. to *selectively* pull things out of the doc comments that are spread across the code base.

That's actually a super nice mechanism as it gives one much freedom to surround the bare generated content with some additional text or move things around to have fine grained control about how things fit into a toc etc.

It's also what trio [uses heavily](https://raw.githubusercontent.com/python-trio/trio/master/docs/source/reference-io.rst) for their documentation.

You probably know all this already but I felt a bit like Jake from Avatar where he suddenly finds himself surrounded by all these glowing stuff fying around him.

I think I'll be able to finish up a PR this week that will already improve a few for the docs :)